openapi/components/schemas/Journal/JournalRecord.yaml

```
@@ -0,0 +1,91 @@
+type: object
+required:
+  - type
+  - journalEntryId
+  - customerId
+  - invoiceId
+  - invoiceItemId
+  - debitAccountId
+  - creditAccountId
+properties:
+  id:
+    $ref: ../ResourceId.yaml
+  journalEntryId:
+    description: ID of the journal entry.
+    readOnly: true
+    type: string
+  customerId:
+    description: ID of the customer.
+    type: string
+  invoiceId:
+    description: ID of the invoice.
+    type: string
+  invoiceItemId:
+    description: ID of the invoice item.
+    type: string
+  type:
+    type: string
+    readOnly: true
+    enum:
+      - automated
+      - manual
+  estimatedAmount:
+    description: |-
+      Amount of revenue estimated to be recognized at the schedule date.
+      Value is ignored when updating a journal record with `type` of `automated`.
+    nullable: true
+    type: number
+    format: double
+  recognizedAmount:
+    description: Amount of revenue recognized at the journal period end.
+    nullable: true
+    type: number
+    format: double
+  debitAccountId:
+    description: ID of the debit journal account.
+    nullable: true
+    type: string
+  creditAccountId:
+    description: ID of the credit journal account.
+    nullable: true
+    type: string
+  createdTime:
+    $ref: ../CreatedTime.yaml
+  updatedTime:
+    $ref: ../UpdatedTime.yaml
+  _embedded:
+    type: object
+    description: Embedded objects that are requested by the `expand` query parameter.
+    readOnly: true
+    properties:
+      customer:
+        $ref: ../Customer.yaml
+      invoice:
+        $ref: ../Invoices/Invoice.yaml
+      invoiceItem:
+        $ref: ../Invoices/InvoiceItem.yaml
+      debitAccount:
+        $ref: ./JournalAccount.yaml
+      creditAccount:
+        $ref: ./JournalAccount.yaml
+  _links:
+    type: array
+    description: Related resource links.
```

nitpick: based on our style
```suggestion
    description: Related links.
```